README.md

```diff
@@ -25,132 +21,112 @@ Package.onUse(function (api) {
-E.g.
+If the target is a directory, it will search for an `index.scss`, `_index.scss`, `index.sass`, or `_index.sass` file inside that directory.
 
-```
-meteor npm install the-package
-cd imports
-ln -s ../node_modules/the-package .
-```
+### Global Include Path
 
-See the [meteor changelog](https://github.com/meteor/meteor/blob/devel/History.md) for more information.
+At the moment, there is no support for a global include path. If there is significant demand, this feature could be added in future updates.
 
-#### Global include path
+### Source Maps
 
-Sometimes a 3rd party module uses import paths that assume that the compiler is
-configured with specific `includePaths` option (e.g. Ionic, Bootstrap, etc.):
-```scss
-@import "ionicons-icons"; // This file is actually placed in another npm module!
-```
+Source maps are enabled by default, helping with debugging by mapping the compiled CSS back to the original Sass files.
+
+### Autoprefixer
+
+To use Autoprefixer, follow the official [PostCSS with Meteor](https://docs.meteor.com/packages/standard-minifier-css.html#standard-minifier-css) documentation.
+
+## Custom Configuration
+
+You can customize the behavior of the Sass compiler by creating a `.scss.config.json` or `scss-config.json` file in the root of your project. This file supports options like `quietDeps`, `verbose`, and others that control Sass compilation behavior.
+
+Example configuration:
 
-Create a configuration file named "`scss-config.json`" at the root of your Meteor
-project to specify include paths that the compiler should use as an extra
-possibility to resolve import paths:
 ```json
 {
-  "includePaths": [
-    "{}/node_modules/ionicons/dist/scss/"
-  ]
+  "quietDeps": true,
+  "verbose": false
 }
 ```
 
+## Additional Features
 
-### Sourcemaps
-These are on by default.
+- **Caching**: The plugin utilizes a caching system to speed up the compilation of Sass files. The default cache size is set to 10 MB.
+- **Import Handling**: When importing Sass files, it attempts to resolve various file extensions (`.scss`, `.sass`, `.css`) and also supports imports from `node_modules` using the `~` syntax.
+- **Partial File Recognition**: Files with an underscore (`_`) prefix are treated as partials and are not compiled into standalone CSS files unless explicitly requested.
 
-### Autoprefixer
-As of Meteor 1.2 autoprefixer should preferably be installed as a separate plugin. You can do so by running:
+### Debug Mode
 
-```
-meteor remove standard-minifiers
-meteor add seba:minifiers-autoprefixer@0.0.2
-```
+You can enable debug mode by setting the `DEBUG_PACKAGE_SASS` environment variable to any value other than `false` or `0`. In debug mode, additional error messages and stack traces will be logged to the console.
 
-In a Meteor 1.3+ project, do the same by running:
-```
-meteor remove standard-minifier-css
-meteor add seba:minifiers-autoprefixer
+```bash
+export DEBUG_PACKAGE_SASS=true
 ```
```

Comment:
@illusionfield I'd suggest keeping short description about prefixer in the main Readme file, for example as [in my PR](https://github.com/Meteor-Community-Packages/meteor-scss/pull/325/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R154-R162)